packages/docs-examples/components/top-menu/top-menu-overflow/top-menu-overflow-example.ts

```diff
@@ -0,0 +1,115 @@
+import { BreakpointObserver } from '@angular/cdk/layout';
+import { CdkScrollable } from '@angular/cdk/overlay';
+import { AsyncPipe } from '@angular/common';
+import { AfterViewInit, ChangeDetectionStrategy, Component, inject, ViewChild } from '@angular/core';
+import { KbqButtonModule, KbqButtonStyles } from '@koobiq/components/button';
+import { KbqComponentColors, PopUpPlacements } from '@koobiq/components/core';
+import { KbqIconModule } from '@koobiq/components/icon';
+import { KbqToolTipModule } from '@koobiq/components/tooltip';
+import { KbqTopMenuModule } from '@koobiq/components/top-menu';
+import { Observable } from 'rxjs';
+import { auditTime, map } from 'rxjs/operators';
+
+/**
+ * @title TopMenu Overflow
+ */
+@Component({
+    standalone: true,
+    selector: 'top-menu-overflow-example',
+    imports: [
+        AsyncPipe,
+        CdkScrollable,
+        KbqTopMenuModule,
+        KbqButtonModule,
+        KbqToolTipModule,
+        KbqIconModule
+    ],
+    changeDetection: ChangeDetectionStrategy.OnPush,
+    template: `
+        @let isDesktopMatches = !!(isDesktop | async)?.matches;
+        <kbq-top-menu [hasOverflow]="hasOverflow | async">
+            <div class="kbq-top-menu-container__left layout-row layout-align-center-center">
+                <div class="layout-row layout-padding-m flex-none">
+                    <i class="layout-row flex" kbq-icon="kbq-dashboard_16"></i>
+                </div>
+                <div class="kbq-title kbq-text-ellipsis">Dashboard</div>
+            </div>
+            <div class="kbq-top-menu__spacer"></div>
+            <div class="kbq-top-menu-container__right layout-row">
+                @for (action of actions; track index; let index = $index) {
+                    <button
+                        [kbqStyle]="action.style || ''"
+                        [color]="action.color || ''"
+                        [kbqTooltipDisabled]="isDesktopMatches"
+                        [kbqPlacement]="PopUpPlacements.Bottom"
+                        [kbqTooltip]="action.title"
+                        kbq-button
+                    >
+                        @if (action.icon) {
+                            <i [class]="action.icon" kbq-icon=""></i>
+                        }
+                        @if (isDesktopMatches) {
+                            {{ action.title }}
+                        }
+                    </button>
+                }
+            </div>
+        </kbq-top-menu>
+        <div class="overflow-content kbq-scrollbar" cdkScrollable>
+            <div style="height: 600px">Scroll down ⬇️ to see box-shadow</div>
+        </div>
+    `,
+    styles: `
+        .kbq-text-ellipsis {
+            white-space: nowrap;
+            overflow: hidden;
+            text-overflow: ellipsis;
+        }
+
+        :host {
+            display: flex;
+            flex-direction: column;
+            height: 400px;
+        }
+
+        .overflow-content {
```

Comment:
```suggestion
        .overflow-content-example {
```
для селекторов внутри примеров, лучше всего добавлять `example` префикс или суффикс, чтобы таким образом отличать селектор примера и реализации